### DISCUSSION TITLE
게시물 작성 시 이미지 등록

### DISCUSSION THREAD

**Discussion by woodimora** [Scope-Team9/Scope_BackEnd#98] [General]
이미지를 우선 S3를 쓰지 않고 서버 자체에 저장하려고 하는데 어떻게 생각하시나요?
나중에 S3 로 변경할 수 있도록 해서 S3가 필요로 하다고 하면 적용하면 좋을 것 같습니다.

S3를 쓴다면 백에서 처리하거나 프론트에서 바로 S3에 접근하는 방법도 있을 것 같습니다.

**Comment by 15011106** [comment]
S3 이용한 파일 저장 구현 전 테스트시 먼저 저장하는 기능이 필요할 것 같습니다.
개인적으로는 속도적인 측면에서는 프론트에서 바로 전달하는것이 빠르지 않을까 싶습니다 !

혹시 마크다운페이지에서 드래그했을 때 특정한 통신 발생하여 로컬 저장이 가능할까요 ?  
사용자가 마크다운으로 파일드래그 -> 서버에서 파일 저장

**Reply by woodimora** [reply]
그건 프론트팀에 물어봐야할 것 같은데 아마 가능할 것으로 보입니다. 
그러나 문제가 있는게 한정적인 용량에서 사용자가 최종적으로 이미지를 업로드하지 않고 게시글 작성을 취소하거나 이미지를 변경했을 때에 사용되지 않는 이미지는 어떻게 처리할 것인가가 고민이 되네요.
